vllm/attention/backends/flashinfer.py

```diff
@@ -135,11 +135,17 @@ def begin_forward(self):
                 return
 
             assert self.prefill_wrapper is not None
+            assert self.query_start_loc is not None
             assert self.paged_kv_indices is not None
             assert self.paged_kv_indptr is not None
             assert self.paged_kv_last_page_len is not None
-            self.paged_kv_indices = self.paged_kv_indices.to(self.device)
```

Comment:
Per https://github.com/flashinfer-ai/flashinfer/issues/362#issuecomment-2261713345, perhaps we should keep this?